packages/data-point/lib/helpers/helpers.js

```diff
@@ -33,13 +33,9 @@ module.exports.resolveEntity = require('../entity-types/base-entity/resolve').re
 function reducify (method) {
   return function () {
     const partialArguments = Array.prototype.slice.call(arguments)
-    return function (acc, done) {
-      const methodArguments = [acc.value].concat(partialArguments)
-      const result = method.apply(null, methodArguments)
-      if (_.isError(result)) {
-        return done(result)
-      }
-      done(null, result)
+    return function (value) {
+      const methodArguments = [value].concat(partialArguments)
+      return method.apply(null, methodArguments)
     }
   }
 }
```

Comment:
will `reducify` still be useful now that value is passed separate from the accumulator?

Reply:
yes because adds a closure to pass the value as the first parameter, which prob should be documented more, but yes still makes sense